pyproject.toml

```diff
@@ -34,6 +34,7 @@ sphinx_rtd_theme = { version = "^0", optional = true }
 sphinxcontrib-programoutput = { version = "^0", optional = true }
 myst-parser = { version = "*", optional = true }
 docutils = { version = ">=0.17", optional = true }
+pyyaml = ">=6.0.1"
```

Comment:
As this adds a new explicit dependency, shouldn't it work just fine by only updating the dependencies without doing this? Looking at the lock file, there is nothing that would prevent and update to the newest pyyaml release in it.

Reply:
It doesn't need to be explicitly pinned

https://github.com/python-kasa/python-kasa/pull/482